src/dataSources/WikibaseActionApi.js

```diff
@@ -24,6 +29,19 @@ module.exports = class WikibaseActionApi extends RESTDataSource {
     return ids.map(id => getEntitiesResponse.entities[id]);
   });
 
+  async getEntitiesByLabel(label) {
+    const getEntitiesResponse = await this.get('', {
+      action: 'wbsearchentities',
+      format: 'json',
+      search: label.queryValue,
+      language: label.queryLang
+    }, {
+      maxBatchSize: 50,
```

Comment:
`maxBatchSize` isn't needed for wbsearchentities because there is no batching for it.

Reply:
oops, copy/pasta